R/empty_assignment_linter.R

```diff
@@ -5,20 +5,18 @@
 #'
 #' @examples
 #' # will produce lints
+#' code_lines <- "x <- {\n}"
```

Comment:
I would revert this one. I have only ever seen this lint show up as `x <- {}`, never with an intervening newline. I think it's more important to represent examples that ppl might actually write / encounter than to avoid multiple lints.